CHANGELOG.md

```diff
@@ -4,6 +4,19 @@
   ## v8.10.0
 
 
+  * **gRPC instrumentation**
+
+    The agent will now instrument [gRPC](https://grpc.io/) activity performed by clients and servers that use the [grpc](https://rubygems.org/gems/grpc) RubyGem. Instrumentation is automatic and enabled by default, so gRPC users should not need to modify any existing application code or agent configuration to benefit from the instrumentation. The instrumentation makes use of New Relic Distributed Tracing for a comprehensive overview of all gRPC traffic taking place across multiple monitored applications. The Ruby agent's use of Distributed Tracing is compatible with other programming language based New Relic agents that support gRPC.
```

Comment:
I'm sorry I didn't catch this last time. Distributed tracing follows a W3C standard, so labeling it as `New Relic` isn't quite appropriate. Infinite tracing, on the other hand, could probably always be prefixed with `New Relic`.
 
```suggestion
    The agent will now instrument [gRPC](https://grpc.io/) activity performed by clients and servers that use the [grpc](https://rubygems.org/gems/grpc) RubyGem. Instrumentation is automatic and enabled by default, so gRPC users should not need to modify any existing application code or agent configuration to benefit from the instrumentation. The instrumentation makes use of distributed tracing for a comprehensive overview of all gRPC traffic taking place across multiple monitored applications. This allows you to observer your client and server activity using any service that adheres to the W3C standard.
```